### PR TITLE
Feat/#8rdhyr #5cbhx2

### PR DESCRIPTION
**Pull request by Kevinjhlin**
Please review my work, I feel like I might not be doing it right

### CODE REVIEW

**Comment by dannylekim** [comment]
Task linked: [CU-5cbhx2 Logout](https://app.clickup.com/t/5cbhx2)

**Comment by dannylekim** [comment]
Task linked: [CU-8rdhyr Home Dashboard v2](https://app.clickup.com/t/8rdhyr)

**Review by asianswagstorm — APPROVED** [review]
I fixed the useInfo redux and updated some previous tests by adding provider.

File: src/views/LoginForm/LoginForm.jsx [hunk 1]
```diff
@@ -216,6 +216,9 @@ class LoginFormContainer extends Component {
         this.handleErrorResponse(response);
       } else {
         localStorage.setItem("billSnap_token", response.token);
+        const {id, ...userProfile} = response.profile;
+        console.log(userProfile);
```
Comment (alphakennyn) [inline src/views/LoginForm/LoginForm.jsx]: please remove

File: src/components/Sidebar/Sidebar.jsx [hunk 2]
```diff
@@ -35,7 +37,13 @@ class Sidebar extends React.Component {
         }
     }
 
+    handleLogoutClick() {
+        localStorage.clear();
```
Comment (alphakennyn) [inline src/components/Sidebar/Sidebar.jsx]: normally we shouldn't clear all of the client browser's local-storage. Just get rid of our token `billsnap-token`

**Review by alphakennyn — CHANGES_REQUESTED** [review]

File: src/components/Sidebar/Sidebar.jsx [hunk 3]
```diff
@@ -35,7 +37,13 @@ class Sidebar extends React.Component {
         }
     }
 
+    handleLogoutClick() {
+        localStorage.removeItem("billSnap_token");
```
Comment (Kevinjhlin) [inline src/components/Sidebar/Sidebar.jsx:41]: @alphakennyn

File: src/views/LoginForm/LoginForm.jsx [hunk 4]
```diff
@@ -216,6 +216,8 @@ class LoginFormContainer extends Component {
         this.handleErrorResponse(response);
       } else {
         localStorage.setItem("billSnap_token", response.token);
+        const {id, ...userProfile} = response.profile;
+        this.props.setUser(userProfile);
```
Comment (Kevinjhlin) [inline src/views/LoginForm/LoginForm.jsx:220]: @alphakennyn Removed console.log

**Comment by dannylekim** [comment]
@alphakennyn  @Kevinjhlin  

If there's no way for us to up that code coverage on the added code, then lmk and I can force merge. But ideally we should be able to hit the code that's been added rather than 80%

**Review by alphakennyn — APPROVED** [review]

**Pull request merged**